content_scripts/model/siteSelectors.js

```diff
@@ -40,6 +40,55 @@ export const siteData = {
     consentCheckbox: {
       selector: 'input[data-test="marketing_consent"]',
       type: 'checkCheckbox'
+    },
+    userName: {
+      selector: 'input[data-test="userName"]',
+      type: 'input',
+      data: fakeDataSb.userName
+    },
+    idNumber: {
+      selector: 'input[data-test="identification_number"]',
+      type: 'input',
+      data: fakeDataSb.idNumber
+    },
+    selectGender: {
+      selectorAll: document.querySelectorAll('[data-test="gender"] > button[data-test="select_button"]')[0],
+      type: 'select',
+      data: 'select_option_genderOptions_female'
+    },
```

Comment:
jak w implementacji selectOption 
to musi być w pełni konfigurowalne i nie dotyczyć tylko z góry określonej struktury